cmd/clusterctl/cmd/restore.go

```diff
@@ -44,6 +44,7 @@ var restoreCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 		return runRestore()
 	},
+	Deprecated: "use 'clusterctl move' instead.",
```

Comment:
nit
```suggestion
	Deprecated: "use 'clusterctl move --from-folder' instead.",
```
same in the docs